docs/source/development/traffic_monitor/traffic_monitor_api.rst

```diff
@@ -343,22 +343,60 @@ TODO
 
 ``/publish/CrStates``
 =====================
-The current state of this CDN per the ref:`health-proto`.
+The current state of this CDN per the :ref:`health-proto`.
 
 ``GET``
 -------
-:Response Type: ?
+:Response Type: Object
+
+.. code-block:: http
+	:caption: Example Request
+
+	GET /publish/CrStates HTTP/1.1
+	Accept: */*
+	Content-Type: application/json
 
 Response Structure
 """"""""""""""""""
+:caches: An object with keys that are the names of monitored :term:`cache servers`
+	:server name: The name of the server being monitored
```

Comment:
I think there needs to be an empty line between indentation levels for this to render properly